Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:20.04 as prebuilder
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && apt-get install -y \
+    git \
+    imagemagick \
+    nodejs \
+    npm \
+    tesseract-ocr-deu \
+    && rm -rf /var/lib/apt/lists/*
+
+# allow ImageMagick to process PDFs
+RUN sed -i '$i\ \ <policy domain="coder" rights="read | write" pattern="PDF" />' /etc/ImageMagick-6/policy.xml
+
+FROM prebuilder as build-console
+
+COPY ./console /usr/local/hub/console
+WORKDIR /usr/local/hub/console
+RUN npm install --no-audit && npm run build
+
+FROM prebuilder as build-server
+
+COPY ./server /home/node/app
+WORKDIR /home/node/app
+
+RUN npm install --no-audit && npm run compile
+
+FROM prebuilder
+COPY --from=build-server /home/node/app/lib /home/node/app
+COPY ./server/config /home/node/app/config
+COPY ./server/public /home/node/app/public
+COPY ./server/package*.json /home/node/app/
+COPY --from=build-console /usr/local/hub/console/dist /home/node/app/ext-console
+
+WORKDIR /home/node/app
+RUN npm install --only=production --no-audit
+
+EXPOSE 3030
+ENV NODE_ENV production
+ENV NODE_CONFIG_ENV docker
+CMD [ "node", "index.js" ]
```

Comment:
The container should be run as the `node` user

Reply:
done